dp-terraform/helm/rhacs-terraform/charts/observability/templates/01-operator-05-subscription.yaml

```
@@ -0,0 +1,14 @@
+﻿apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
+    operators.coreos.com/observability-operator.rhacs-observability: ''
+  name: observability-operator
+  namespace: {{ include "observability.namespace" . }}
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: observability-operator
+  source: observability-operator-manifests
+  sourceNamespace: rhacs-observability
```

I think we should use a namespace where `CatalogSource` is defined, and in our case, it's a variable.
```suggestion
  sourceNamespace: {{ include "observability.namespace" . }}
```